### PR TITLE
feat(validator): add validator metrics

### PR DESCRIPTION
**Pull request by Pangssu**
Added metric
1. The deposit amount into the ValidatorPool contract.
2. The address of the next validator.
3. The output index that the challenge function last checked.

**If there's a metric you need, feel free to leave it in the issue.**

### CODE REVIEW

**Review by chokobole — APPROVED** [review]
LGTM

File: components/validator/metrics/metrics.go [hunk 1]
```diff
@@ -74,6 +82,28 @@ func NewMetrics(procName string) *Metrics {
 			Name:      "up",
 			Help:      "1 if the kroma-validator has finished starting up",
 		}),
+		Nonce: factory.NewGauge(prometheus.GaugeOpts{
+			Namespace: ns,
+			Name:      "nonce",
+			Help:      "",
```
Comment (seolaoh) [inline components/validator/metrics/metrics.go:88]: Why is it an empty string?

**Review by 0xHansLee — APPROVED** [review]
lgtm

**Review by kangsorang — APPROVED** [review]
lgtm

**Review by seolaoh — APPROVED** [review]
LGTM

**Pull request merged**